incubator/opentelemetry-sampler-aws-xray/README.md

```diff
@@ -32,19 +29,19 @@ const _tracerConfig = {
 }
 
 const sdk = new opentelemetry.NodeSDK({
-        textMapPropagator: new AWSXRayPropagator(),
-        instrumentations: [
-            new HttpInstrumentation(),
-            new AwsInstrumentation({
-                suppressInternalInstrumentation: true
-            }),
-        ],
-        resource: _resource,
-        spanProcessor: _spanProcessor,
-        traceExporter: _traceExporter,
-    });
-
-    sdk.configureTracerProvider(_tracerConfig, _spanProcessor);
+    serviceName: "remote-sampler-app",
+    textMapPropagator: new AWSXRayPropagator(),
+    instrumentations: [
+        new HttpInstrumentation(),
+        new AwsInstrumentation({
+            suppressInternalInstrumentation: true
+        }),
+    ],
+    spanProcessor: _spanProcessor,
+    traceExporter: _traceExporter,
+});
+
+sdk.configureTracerProvider(_tracerConfig, _spanProcessor);
```

Comment:
Reviewer note: This just drops the `Resource` usage in favour of the `serviceName` config option to `NodeSDK`. It also cleans up indentation. https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2210/files?w=1 (without whitespace) shows a smaller diff.